addon/HomepageStudy.jsm

```diff
@@ -71,19 +76,77 @@ function getHomepage() {
   return eTLD;
 }
 
+/**
+ * Returns the correct parameters depending if it's a standalone execution or a simulation.
+ * @param {boolean} isSimulation - Boolean indicating if the addon is run for a simulation.
+ * @param {string} rapporPath  - Path of the RAPPOR simulator.
```

Comment:
nit: it's missing the documentation for instance.

Reply:
Done